nucliadb_dataset/nucliadb_dataset/__init__.py

```diff
@@ -37,3 +43,6 @@ class DatasetType(str, Enum):
 class ExportType(str, Enum):
     DATASETS = "DATASETS"
     FILESYSTEM = "FILESYSTEM"
+
+
+__all__ = ("NucliaDBDataset", "NucliaCloudDataset", "Task", "download_all_partitions")
```

Comment:
If we use `__all__`, we need to include other imports here.
```suggestion
__all__ = ("NucliaDBDataset", "NucliaCloudDataset", "Task", "download_all_partitions", "NUCLIA_GLOBAL", "CLIENT_ID", "DatasetType", "ExportType")
```

What I like to do is make sure that the package module root only ever imports from other places and maybe internal code never imports from it. This prevents circular import issues.

Reply:
good catch. thanks! fixed it